ibft2-eks/.kube/config.template

```diff
@@ -0,0 +1,30 @@
+apiVersion: v1
+clusters:
+- cluster:
+    server: <INSERT THE CLUSTER ENDPOINT FROM THE CFN OUTPUTS>
+    certificate-authority-data: <INSERT THE CA DATA FROM THE EKS CONSOLE>
```

Comment:
Would it be possible or desirable to have a script to get these values using the AWS-CLI?

Reply:
done, using a script now and have updated the readme